css/base/_colors.scss

```diff
@@ -0,0 +1,8 @@
+$color-white: #ffffff;
+$color-black: #000000;
+$color-primary: #dd2a7c;
+$color-link: #3095f6;
+$color-secondary: #fafafa;
+$color-search: #efefef;
```

Comment:
nazwa koloru powinna byc bardziej uniwersalna

okreslajac nazwe koloru jako "color-search" albo "color-border" praktycznie zamykasz sobie mozliwosc uzycia go gdzie kolwiek indziej

podsylam artykul / sciage: 
https://css-tricks.com/what-do-you-name-color-variables/

warto zwrocic uwage na PDF z kolorami jaki wam wyslalem do shopstagrama, zobaczysz tam ze 3 albo 4 z nich sa roznymi odcieniami szarosci, wiec mozna je np nazwac

color-neutral
color-neutral--light
color-neutral--dark

albo nawet w ostatecznosci

color-gray
color-gray--light
color-gray--dark